app/controllers/AddBusinessNameController.scala

```diff
@@ -117,7 +120,10 @@ class AddBusinessNameController @Inject()(authenticate: AuthenticationPredicate,
               },
               formData => {
                 Future {
-                  Redirect(routes.AddBusinessStartDate.showAgent())
+                  //Should be the below comment, but changed to redirect to business trade so ATs can be made while Start Date is not yet implemented
```

Comment:
same here, wrap Redirect with Future.complete and remove any commented code across your PR

Reply:
Addressed